etc/schema.json

```diff
@@ -1156,6 +1171,15 @@
                         "opcode": {
                             "description": "DNS opcode as an integer",
                             "type": "integer"
+                        },
+                        "aa": {
+                            "type": "boolean"
```

Comment:
@jasonish should we add description for these added fields ?

Reply:
`aa` is already present !

Reply:
> @jasonish should we add description for these added fields ?

Yes, will help with documentation generation.